java/src/jmri/jmrit/operations/locations/gui/Bundle.java

```diff
@@ -0,0 +1,97 @@
+package jmri.jmrit.operations.locations.gui;
+
+import edu.umd.cs.findbugs.annotations.SuppressFBWarnings;
+import java.util.Locale;
+import javax.annotation.CheckReturnValue;
+import javax.annotation.CheckForNull;
+import javax.annotation.ParametersAreNonnullByDefault;
+
+@ParametersAreNonnullByDefault
+@CheckReturnValue
+@SuppressFBWarnings(value = "NM_SAME_SIMPLE_NAME_AS_SUPERCLASS", justification = "Desired pattern is repeated class names with package-level access to members")
+
+@javax.annotation.concurrent.Immutable
+
+/**
+ * Provides standard access for resource bundles in a package.
+ *
+ * Convention is to provide a subclass of this name in each package, working off
+ * the local resource bundle name.
+ *
+ * @author Bob Jacobsen Copyright (C) 2012
+ * @since 3.3.1
+ */
+public class Bundle extends jmri.jmrit.operations.Bundle {
+
+    @CheckForNull
+    private static final String name = "jmri.jmrit.operations.locations.JmritOperationsLocationsBundle"; // NOI18N
```

Comment:
Is this correct? The class is in the gui package but it refers to a bundle one level up.

If this package doesn't need its own bundle, set the name to `null`.

Reply:
I just copied the bundle from the previous level not looking at the content. I thought they were all the same.  I'll fix.

Reply:
Setting to null causes failures.

Reply:
Ensure that the Bundle class inherits the Bundle class one package up. And that that Bundle has the correct name